src/sentry/eventstream/base.py

```diff
@@ -187,7 +187,7 @@ def tombstone_events_unsafe(
         self,
         project_id: int,
         event_ids: Sequence[str],
-        old_primary_hash: Union[str, bool] = False,
```

Comment:
lol. I ran git blame and wondered who tf decided that `old_primary_hash` should be a boolean... turns out I decided that, and you just added the type annotation. `None` is indeed more correct.